starter-code/src/app/app.component.html

```diff
@@ -1,3 +1,9 @@
-<h1>
-  {{title}}
-</h1>
+<div class="container-fluid">
+  <header>
+  </header>
+  <nav class="index-container">  
+      <a routerLink="/"><img src="../assets/images/ironhack-france.png" alt=""></a>
```

Comment:
This is wrong syntax for the router link directive, should be:
`[routerLink]=...`